controllers/storagecluster/storageclasses.go

```diff
@@ -271,5 +281,15 @@ func (r *StorageClusterReconciler) newStorageClassConfigurations(initData *ocsv1
 	if initData.Spec.ExternalStorage.Enable || err == nil && !skip {
 		ret = append(ret, newCephOBCStorageClassConfiguration(initData))
 	}
+	// encrypted Ceph Block Pool storageclass will be returned only if
+	// storage-class-wide encryption + kms is enabled and KMS ConfigMap is available
+	if initData.Spec.Encryption.StorageClassWide && initData.Spec.Encryption.KeyManagementService.Enable {
+		kmsConfig, err := getKMSConfigMap(KMSConfigMapName, initData, r.Client)
+		if err == nil && kmsConfig != nil {
+			serviceName := kmsConfig.Data["KMS_SERVICE_NAME"]
```

Comment:
need to check once with Rook and Ceph-CSI whether they are willing to change this key from`KMS_SERVICE_NAME` to `KMS_CONNECTION_NAME`

Reply:
As we discussed ```KMS_CONNECTION_NAME``` is not required

Reply:
We better log the err with clear message

Reply:
ack...

Reply:
added on L292